cylc/flow/scripts/lint.py

```diff
@@ -19,11 +19,9 @@
 Checks code style, deprecated syntax and other issues.
 
 """
-# NOTE: docstring needed for `cylc help all` output
+# NOTE: docstring needed for `cylc help all` output and docs
 # (if editing check this still comes out as expected)
 
-LINT_SECTIONS = ['cylc-lint', 'cylclint', 'cylc_lint']
```

Comment:
Using `[cylclint]` or `[cylc_lint]` resulted in traceback so I've gone ahead and removed those

Reply:
Massive violation of [PEP20.15](https://github.com/python/cpython/blob/5e6482806482af364ec93823893a49767e5f786f/Lib/this.py#L15) - yes I know I wrote it...